osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -191,8 +191,6 @@ private void validateOnlineIds(BeatmapSetInfo beatmapSet)
         {
             var beatmapIds = beatmapSet.Beatmaps.Where(b => b.OnlineBeatmapID.HasValue).Select(b => b.OnlineBeatmapID).ToList();
 
-            LogForModel(beatmapSet, $"Validating online IDs for {beatmapSet.Beatmaps.Count} beatmaps...");
```

Comment:
Was this meant to be in this diff? It's not in the description and I don't see anything wrong with this log statement.

Reply:
Yep. It sounds like it's doing something online but isn't. Also it's meaningless in 99% of cases (it still outputs if it makes a change).